fontforgeexe/contextchain.c

```diff
@@ -30,6 +30,7 @@
 #include "asmfpst.h"
 #include "chardata.h"
 #include "fontforgeui.h"
+#include "../gdraw/ggadgetP.h"
```

Comment:
Ugh, including this private header is pretty gross, since we're touching this can we just add a gdraw method to set the default font?

Reply:
So this would become `GDrawSetDefaultFont(ccd->glyphs_simple)`?  That seems reasonable.

Reply:
Sounds good to me

Reply:
Done